app/helpers/flipper_helper.rb

```diff
@@ -2,4 +2,20 @@ module FlipperHelper
   def resolve_use_who_standard(use_who_standard)
     use_who_standard.nil? ? Flipper.enabled?(:diabetes_who_standard_indicator) : use_who_standard
   end
+
+  def all_district_overview_feature_enabled?
+    Flipper.enabled?(:all_district_overview, current_admin)
+  end
+
+  def all_district_overview_enabled?
+    all_district_overview_feature_enabled? && params[:facility_group] == "all-districts"
```

Comment:
`params` in the helper is sort of an antipattern since this is meant to be in the controller which terminates the request. 

Please refactor this